cjworkbench/i18n/templates.py

```diff
@@ -8,9 +8,17 @@ def context_processor(request) -> dict:
 
 
 def get_i18n_context(locale=None, user=None) -> dict:
+    current_locale_id = locale or getattr(user, "locale_id", default_locale)
     return {
         "i18n": {
-            "locale_id": locale or getattr(user, "locale_id", default_locale),
+            "locale_id": current_locale_id,
             "show_switcher": getattr(user, "is_staff", False) if user else False,
+            "locales_data": [
+                {
+                    "locale_id": locale_id,
```

Comment:
This object's keys should be `"id"` and `"name"`.

I know that in `serializers.py` we currently put understores in some JS Object keys. That's always been a mistake. (It's hard to correct it after the fact.) Let's not introduce more :).